bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -50,31 +45,105 @@ spec:
         kind: CommonService
         name: commonservices.operator.ibm.com
         specDescriptors:
-          - description: 'Accepting the license - URL: https://ibm.biz/icpfs39license'
+          - displayName: License
+            path: license
+            description: License information for this instance. You must accept the license.
+          - description: Read and accept the license that is applicable to your installation. For more information, see https://ibm.biz/icpfs39license
             displayName: Accept
             path: license.accept
             value:
               - false
             x-descriptors:
               - urn:alm:descriptor:com.tectonic.ui:checkbox
-          - description: The size of deployment profile for IBM Cloud Pak foundational services
+          - path: license.license
+            x-descriptors:
+              - urn:alm:descriptor:com.tectonic.ui:hidden
+          - path: license.use
+            x-descriptors:
+              - urn:alm:descriptor:com.tectonic.ui:hidden
+          - path: license.key
+            x-descriptors:
+              - urn:alm:descriptor:com.tectonic.ui:hidden
+          - path: license.use
+            x-descriptors:
+              - urn:alm:descriptor:com.tectonic.ui:hidden
+          - description: The size of deployment profile for IBM Cloud Pak foundational
```

Comment:
IBM Cloud Pak Foundational Services (upper case)

Reply:
Will keep as lowercase for consistency with the UI for now